relay-server/src/actors/spooler/mod.rs

```diff
@@ -636,17 +727,51 @@ impl BufferState {
                         "failed to spool the in-memory buffer to disk",
                     );
                 }
+                relay_log::trace!(
+                    "Transition to disk spool: # of envelopes = {}",
```

Comment:
How is this number of envelopes calculated? IIRC it was an estimation and not a super accurate number, I think we should note that in the trace message.

Reply:
This number here should be correct